operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -246,16 +176,8 @@ When triaging a bug report, consider the following:
 
 **Q: Will RC testing remain, or will RC testing process change?**
```

Comment:
I think we could remove this section about RC testing entirely; the old RC team-testing process has been gone for some time, and any remaining manual testing is already mentioned / covered in other docs, Zephyr test cycles, and playbooks.